src/main.rs

```diff
@@ -237,10 +236,67 @@ fn parse_matches() -> clap::ArgMatches {
                 .takes_value(true)
                 .default_value("0")
                 .help("Number of rpc nodes")
-                .validator(|s| match s.parse::<i32>() {
-                    Ok(n) if n >= 0 => Ok(()),
-                    _ => Err(String::from("number_of_rpc_nodes should be >= 0")),
-                }),
+        )
+        // Client Config
+        .arg(
+            Arg::with_name("number_of_clients")
+                .long("num-clients")
+                .short('c')
+                .takes_value(true)
+                .default_value("0")
+                .help("Number of clients ")
+        )
+        .arg(
+            Arg::with_name("client_type")
+                .long("client-type")
+                .takes_value(true)
+                .default_value("tpu-client")
+                .possible_values(["tpu-client", "rpc-client"])
+                .help("Client Config. Set Client Type"),
+        )
+        .arg(
+            Arg::with_name("client_to_run")
+                .long("client-to-run")
+                .takes_value(true)
+                .default_value("bench-tps")
+                .possible_values(["bench-tps", "idle"])
+                .help("Client Config. Set Client to run"),
+        )
+        .arg(
+            Arg::with_name("bench_tps_args")
+                .long("bench-tps-args")
+                .value_name("KEY VALUE")
+                .takes_value(true)
+                .multiple(true)
+                .number_of_values(1)
+                .help("Client Config.
+                User can optionally provide extraArgs that are transparently
+                supplied to the client program as command line parameters.
+                For example,
+                    --bench-tps-args 'tx-count=5000 thread-batch-sleep-ms=250'
+                This will start bench-tps clients, and supply '--tx-count 5000 --thread-batch-sleep-ms 250'
+                to the bench-tps client."),
+        )
+        .arg(
+            Arg::with_name("target_node")
+                .long("target-node")
+                .takes_value(true)
+                .help("Client Config. Optional: Specify an exact node to send transactions to. use: --target-node <Pubkey>.
+                Not supported yet. TODO..."),
+        )
+        .arg(
+            Arg::with_name("duration")
+                .long("duration")
+                .takes_value(true)
+                .default_value("7500")
+                .help("Client Config. Seconds to run benchmark, then exit; default is forever use: --duration <SECS>"),
+        )
+        .arg(
+            Arg::with_name("num_nodes")
+                .long("num-nodes")
+                .short('N')
+                .takes_value(true)
+                .help("Client Config. Optional: Wait for NUM nodes to converge: --num-nodes <NUM> "),
```

Comment:
with `.value_name("NUM")` you can also simplify this help text
```suggestion
                .help("Client Config. Optional: Wait for NUM nodes to converge"),
```